sui/p2p_transfer/index.js

```diff
@@ -36,31 +38,33 @@ const main = async () => {
       const txb = new TransactionBlock();
       const [coin] = txb.splitCoins(txb.gas, [txb.pure(1)]);
       txb.transferObjects([coin], receiver_address);
+      txb.setSender(sender_keypair.toSuiAddress());
+      txb.setGasBudget(5_000_000)
+
+      const buildStartTime = performance.now();
+      const bytes = await txb.build({ client: suiClient });
 
       const startTime = performance.now();
-      const transfer_resp = await suiClient.signAndExecuteTransactionBlock({signer: sender_keypair, transactionBlock: txb, 	options: {
-          showBalanceChanges: true,
-          showEffects: true,
-          showEvents: true,
-          showInput: true,
-          showObjectChanges: true,
-          showRawInput: true,
-      },});
-      const wait_resp = await suiClient.waitForTransactionBlock({ digest: transfer_resp.digest, options: {
+      await suiClient.signAndExecuteTransactionBlock({signer: sender_keypair, transactionBlock: bytes, options: {
           showBalanceChanges: true,
           showEffects: true,
           showEvents: true,
           showInput: true,
```

Comment:
This is definitely over selecting, I'm not sure why this had all these options enabled.  It would probably make sense to mirror something similar to what the Aptos SDK is selecting (where applicable)